src/Elastic.Markdown/Diagnostics/DiagnosticsChannel.cs

```diff
@@ -124,6 +124,7 @@ protected virtual void HandleItem(Diagnostic diagnostic) { }
 
 	public virtual async Task StopAsync(CancellationToken cancellationToken)
 	{
+		Console.WriteLine("Stopping diagnostics");
```

Comment:
Should this be a logger?

Or is it a left over from debugging?

Reply:
Indeed a leftover 😢 thanks for catching it ammended the PR.